src/Renderer/PhpRenderer.php

```diff
@@ -414,7 +429,7 @@ public function setFilterChain(FilterChain $filters)
     public function getFilterChain()
     {
         if (null === $this->__filterChain) {
-            $this->setFilterChain(new FilterChain());
+            $this->__filterChain = new FilterChain();
```

Comment:
Similar to above location: setter may have been overridden, although I'll let this one through.